openapi/code/named.go

```diff
@@ -208,6 +210,8 @@ func (n *Named) Summary() string {
 	return ""
 }
 
+var markdownLink = regexp.MustCompile(`(?m)\[([^\]]+)\]\(([^\)]+)\)`)
```

Comment:
AFAIK `?m` only applies to `^` and `$` and you need `?s` to match newlines. Can you add a comment how you expect this to behave w.r.t. newlines?